sw/host/opentitanlib/src/transport/hyperdebug/mod.rs

```diff
@@ -572,6 +572,17 @@ impl Inner {
     }
 }
 
+/// Internal state of the Hyperdebug struct, this struct is reference counted such that Gpio,
```

Comment:
This comment is no longer accurate.  As far as I can tell, nothing other than the main `Hyperdebug` struct will have reference to this new `Inner` struct, so there is no point in using `Rc<Inner>` up there.  In fact, I think that the five fields here should just become direct members of the `Hyperdebug` struct.